app/assets/javascripts/Components/Result/url_viewer.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "../../../stylesheets/common/_url_viewer.scss";
```

Comment:
Don't import css files in jsx. Import it in the same we do do for the `_file_viewer.scss` file (in `_markus.scss)